packages/dbml-core/src/export/DbmlExporter.js

```diff
@@ -15,8 +15,24 @@ class DbmlExporter {
     return /\s*(\*|\+|-|\([A-Za-z0-9_]+\)|\(\))/g.test(str);
   }
 
-  static escapeNote (str) {
-    return str.replaceAll("'", "\\'")
+  static escapeNote (str, indent) {
+    if (str === null) {
+      return ''
+    }
+    if (str.match(/[\n\r']/)) {
+      const closure = function() {
+        function compute_line(line, lineNumber){
+          var prefix = (lineNumber > 0) ? ' '.repeat(indent) : ""; // We add 3 because 3 simple quotes
+          return prefix + line;
+        }
+        return compute_line
+      }
+      const comp_line = closure();
+      const res = str.split(/[\r\n]/).map(comp_line).join("\n");
+      return `'''${res}'''`;
+    } else {
+      return `'${str}'`
+    }
   }
```

Comment:
* Since indentation is not yet used, I suggest to remove it to keep the code clean
* Should add back the escaping behavior, otherwise, we will, have to handle the edge case of ''' presented in the source text could break our note as well.
  * Beside that, in the old code, I found an edge case where if the input contain `\'`, our note will break
* In windows, \r\n (CRLF) is viewed as 1 line break, therefore, we should treat it as 1 line break only. 

```suggestion
  static escapeNote (str) {
    if (str === null) {
      return '';
    }

    // edge case: source text contain both \', we can preserve that by escaping any \ character
    let newStr = str.replaceAll('\\', '\\\\');
    newStr = newStr.replaceAll("'", "\\'");

    if (newStr.match(/[\n\r']/)) {
      newStr = newStr.replaceAll('\r\n', '\n'); // turn all CRLF to LF
      return `'''${newStr}'''`;
    }

    return `'${newStr}'`;
  }
```